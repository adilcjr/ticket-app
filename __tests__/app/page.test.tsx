import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Dashboard from "@/app/page";

describe("Page", () => {
  test("renders a heading", () => {
    render(<Dashboard />);

    const heading = screen.getByRole("heading", { level: 1 });

    expect(heading).toBeInTheDocument();
  });

  test("renders homepage unchanged", () => {
    const { container } = render(<Dashboard />);
    expect(container).toMatchSnapshot();
  });
});
