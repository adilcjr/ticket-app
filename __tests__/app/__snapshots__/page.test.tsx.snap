// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders homepage unchanged 1`] = `
<div>
  <div>
    <h1>
      Dashboard
    </h1>
  </div>
</div>
`;
